data/src/main/java/com/example/data/CustomerDataApplication.java

```diff
@@ -0,0 +1,51 @@
+package com.example.data;
+
+/**
+ * Create app that should support create, read, update and delete operations for some entity
+ * Use Spring Data module
+ * Don't use Spring Data REST starter
+ */
+
+import com.example.data.backendapi.dto.Customer;
+import com.example.data.backendapi.dao.CustomerRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class CustomerDataApplication {
+
+	private static final Logger log = LoggerFactory.getLogger(CustomerDataApplication.class);
+
+	public static void main(String[] args) {
+		SpringApplication.run(CustomerDataApplication.class);
+	}
+
+	//TODO: the demo method will be removed
+	@Bean
+	public CommandLineRunner demo(CustomerRepository repository) {
```

Comment:
Please move the bean into the separate own file or some Configuration bean.
Keep Application bean simple (just main function)
All other stuff like bean definitions, functional and feature annotations, console runners etc keep in the appropriated files (separate or inside Configuration bean)